src/main/res/values/strings.xml

```diff
@@ -216,20 +216,20 @@
     <string name="preference_title_about_rating">Rate</string>
     <string name="preference_title_about_version">Version</string>
     <string name="preference_title_about_intro">Intro</string>
-    <string name="preference_title_account_sign_out">Sign out</string>
+    <string name="preference_title_account_sign_out">Sign Out</string>
     <string name="preference_title_server_connection">Connection</string>
-    <string name="preference_title_tell_a_friend">Tell a friend</string>
+    <string name="preference_title_tell_a_friend">Tell a Friend</string>
     <string name="preference_title_upload_settings">Auto Upload Photos</string>
     <string name="preference_title_language">Language</string>
 
-    <string name="preference_title_upload_server">Select server</string>
-    <string name="preference_title_upload_share">Select share</string>
-    <string name="preference_title_upload_location">Upload location</string>
-    <string name="preference_title_upload_data">Allow on cellular data</string>
+    <string name="preference_title_upload_server">Select Server</string>
+    <string name="preference_title_upload_share">Select Share</string>
+    <string name="preference_title_upload_location">Upload Location</string>
+    <string name="preference_title_upload_data">Allow on Cellular Data</string>
 
-    <string name="preference_summary_server">Select Server</string>
-    <string name="preference_summary_share">Select Share</string>
-    <string name="preference_summary_location">Select Location</string>
+    <string name="preference_summary_server">Select server</string>
```

Comment:
Please follow either sentence case for all sentences or capitalization of the first letter of each word but please don't use both strategies.